Source/source/GPSMessageInterface.hpp

```diff
@@ -0,0 +1,20 @@
+#ifndef LORIS_GPS_MESSAGE_INTERFACE
+#define LORIS_GPS_MESSAGE_INTERFACE
+
+#define INDENT_SPACES "  "
+#include "Message.h"
+#include "MessageBuilder.h"
+#include "GPS_Data_Types.h"
+#include <string>
+#include <iostream>
+#include <math.h>
+#include <chrono>
+#include "Identifiers.h"
+
+using namespace std;
+//#include "MessageSenderInterface.h"
+
+bool send_message(gps_data* decoded_data);
+Message get_message();
+
```

Comment:
You should declare the "decode_time_from_int" method here

Reply:
Fixed this